addon/mixins/json-api-bagaaravel-serializer.js

```diff
@@ -28,8 +28,9 @@ export default Mixin.create({
   // Bagaaravel only allows hasMany relationships to be saved via the record when it's new.
   shouldSerializeHasMany (snapshot) {
     let isRecordNew = snapshot.record.isNew
+    let isRecordSaving = snapshot.record.isSaving
```

Comment:
PP: To make it sound a tiny bit nicer you could rename it to `isRecordBeingSaved`. But not a requirement at all.